applications/MappingApplication/tests/cpp_tests/test_nearest_neighbor_aux_classes.cpp

```diff
@@ -146,9 +146,9 @@ KRATOS_TEST_CASE_IN_SUITE(NearestNeighborInterfaceInfo_MatchingNeighborFound, Kr
 
     KRATOS_CHECK(nearest_neighbor_info.GetLocalSearchWasSuccessful());
 
-    int found_id;
+    std::vector<int> found_id(1);
     nearest_neighbor_info.GetValue(found_id, MapperInterfaceInfo::InfoType::Dummy);
-    KRATOS_CHECK_EQUAL(found_id, expected_id_found);
+    KRATOS_CHECK_EQUAL(found_id[0], expected_id_found);
```

Comment:
can you please extend this test to make sure both found nodes are tested?
I.e. test the content of found_id and also the weights

then it is ready, sorry that I didn't get to it yet :/

Reply:
Test added